kornia/__init__.py

```diff
@@ -51,6 +51,7 @@
     zca_whiten,
 )
 from kornia.feature import gftt_response, harris_response, hessian_response, MKDDescriptor, nms2d, SIFTDescriptor
+from kornia.feature import DeFMO
```

Comment:
@ducha-aiki I'd keep it under `kornia.feature` instead of top level. Probably we need to review all the namespaces organization at some point.

Reply:
Well, it does not really fit into the (local) feature module, tbh and because of being high-level application, I think would benefit from being exposed. 
And yes, we need to review and define some policy